bin/torii/src/main.rs

```diff
@@ -130,6 +130,12 @@ async fn main() -> anyhow::Result<()> {
         .connect_with(options)
         .await?;
 
+    // Disable auto-vacuum
+    sqlx::query("PRAGMA auto_vacuum = NONE;").execute(&pool).await?;
```

Comment:
do we want to always disable these or only when it is running in memory?

Reply:
You are right I will set it only when in memory